### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by pieandcakes**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/language-service.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Additional context**
+*Call Stacks: For bugs like crashes, deadlocks, infinite loops, etc. that we are not able to repro and for which the call stack may be useful, please attach a debugger and/or create a dmp and provide the call stacks. Starting with 0.17.3, Windows binaries have symbols available in VS Code by setting your "symbolSearchPath" to "http://msdl.microsoft.com/download/symbols".*
+Add any other context about the problem here including log messages in your Output window ("C_Cpp.loggingLevel": "Information" in settings.json).
```
Comment (bobbrow) [inline .github/ISSUE_TEMPLATE/language-service.md]: May want to list "Debug" instead of "Information" for the logging level

**Comment by bobbrow** [comment]
I'm confused as to what the difference is between [tag]-issue.md and [tag].md is.  They look like duplicates of each other.  Is there a reason for this that I just don't understand?  Is one supposed to be a feature request and the other a bug?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
```
Comment (bobbrow) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Smartphone?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
```
Comment (bobbrow) [inline .github/ISSUE_TEMPLATE/bug_report.md]: I don't think Browser matters

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (bobbrow) [reply .github/ISSUE_TEMPLATE/bug_report.md]: It seems there is a lot of boilerplate left in these templates.  Can you remove the stuff that doesn't apply?

**Review by bobbrow — CHANGES_REQUESTED** [review]

**Comment by pieandcakes** [comment]
@bobbrow I'll see what i can do. I used their tool within the repo and it created the templates.

File: .github/ISSUE_TEMPLATE/language-service.md (hunk 1 above)
Comment (pieandcakes) [reply .github/ISSUE_TEMPLATE/language-service.md]: I pulled it out of the current template. Is it supposed to be "debug" ?

**Comment by bobbrow** [comment]
Are we going to add a feature request template too?

**Comment by pieandcakes** [comment]
@bobbrow Most of our feature requests come from bug reports, which will filter to which section that feature will be for. I would suggest starting with an Issue template and marking it with a label as a feature request.

File: .github/ISSUE_TEMPLATE/general-extension.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
```
Comment (bobbrow) [inline .github/ISSUE_TEMPLATE/general-extension.md]: remove smartphone section

File: .github/ISSUE_TEMPLATE/general-extension.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```
Comment (bobbrow) [inline .github/ISSUE_TEMPLATE/general-extension.md]: remove desktop section (duplicate info as the Describe section)

**Review by WardenGnaw — APPROVED** [review]

**Review by bobbrow — CHANGES_REQUESTED** [review]
general template has some stuff that should be removed.

**Comment by WardenGnaw** [comment]
With the new templates, I would remove ISSUE_TEMPLATE.md

**Review by bobbrow — APPROVED** [review]
Thanks for doing this!

**Pull request merged**